src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.lang.StringBuilder;
+
+/** Todo inherits from task. Only has a description. */
+public class Todo extends Task {
+  String description;
+
+  Todo(String input) {
+    super(input);
+    this.description = getDescription(input);
+  }
+
+  protected String getDescription(String input) {
+    StringBuilder str = new StringBuilder();
+    String[] strArr = input.split(" ");
+    for (int i = 1; i < strArr.length; i++) {
+      str.append(strArr[i]).append(" ");
+    }
+    return str.toString();
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Might be more concise to not use StringBuilder here and simply return a concatenated string. (e.g. return "[T]" + this.getStatusIcon() + ...)

But if you wish to maintain uniformity of string manipulation with your codebase (since you use it in many places), could just stick with the StringBuilder.